## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 421308`
Commit: `4fa8ad71bfa331734f20abc0aa9e9a366fff33b7`

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 38 packages marked as broken and skipped:</summary>
  <ul>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.catboost</li>
    <li>python312Packages.catboost.dist</li>
    <li>python312Packages.kokoro</li>
    <li>python312Packages.kokoro.dist</li>
    <li>python312Packages.lida</li>
    <li>python312Packages.lida.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.orange3</li>
    <li>python312Packages.orange3.dist</li>
    <li>python312Packages.plopp</li>
    <li>python312Packages.plopp.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python312Packages.skytemple-dtef</li>
    <li>python312Packages.skytemple-dtef.dist</li>
    <li>python312Packages.skytemple-files</li>
    <li>python312Packages.skytemple-files.dist</li>
    <li>python313Packages.catboost</li>
    <li>python313Packages.catboost.dist</li>
    <li>python313Packages.kokoro</li>
    <li>python313Packages.kokoro.dist</li>
    <li>python313Packages.lida</li>
    <li>python313Packages.lida.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.orange3</li>
    <li>python313Packages.orange3.dist</li>
    <li>python313Packages.plopp</li>
    <li>python313Packages.plopp.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
    <li>python313Packages.skytemple-dtef</li>
    <li>python313Packages.skytemple-dtef.dist</li>
    <li>python313Packages.skytemple-files</li>
    <li>python313Packages.skytemple-files.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 109 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>beets</li>
    <li>beets-unstable</li>
    <li>beets-unstable.dist</li>
    <li>beets-unstable.doc</li>
    <li>beets-unstable.man</li>
    <li>beets.dist</li>
    <li>beets.doc</li>
    <li>beets.man</li>
    <li>jadx</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.fastai</li>
    <li>python312Packages.fastai.dist</li>
    <li>python312Packages.k-diffusion</li>
    <li>python312Packages.k-diffusion.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.optuna-dashboard</li>
    <li>python312Packages.optuna-dashboard.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.spacy</li>
    <li>python312Packages.spacy-curated-transformers</li>
    <li>python312Packages.spacy-curated-transformers.dist</li>
    <li>python312Packages.spacy-loggers</li>
    <li>python312Packages.spacy-loggers.dist</li>
    <li>python312Packages.spacy-lookups-data</li>
    <li>python312Packages.spacy-lookups-data.dist</li>
    <li>python312Packages.spacy-transformers</li>
    <li>python312Packages.spacy-transformers.dist</li>
    <li>python312Packages.spacy.dist</li>
    <li>python312Packages.textacy</li>
    <li>python312Packages.textacy.dist</li>
    <li>python312Packages.textnets</li>
    <li>python312Packages.textnets.dist</li>
    <li>python312Packages.wandb</li>
    <li>python312Packages.wandb.dist</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.optuna-dashboard.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 242 packages built:</summary>
  <ul>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>cve-bin-tool</li>
    <li>cve-bin-tool.dist</li>
    <li>das</li>
    <li>das.dist</li>
    <li>deeptools</li>
    <li>deeptools.dist</li>
    <li>flattenReferencesGraph</li>
    <li>flattenReferencesGraph.dist</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>pianotrans</li>
    <li>pianotrans.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python312Packages.atopile</li>
    <li>python312Packages.atopile.dist</li>
    <li>python312Packages.ax-platform</li>
    <li>python312Packages.ax-platform.dist</li>
    <li>python312Packages.chart-studio</li>
    <li>python312Packages.chart-studio.dist</li>
    <li>python312Packages.compressai</li>
    <li>python312Packages.compressai.dist</li>
    <li>python312Packages.dash</li>
    <li>python312Packages.dash-bootstrap-components</li>
    <li>python312Packages.dash-bootstrap-components.dist</li>
    <li>python312Packages.dash-bootstrap-templates</li>
    <li>python312Packages.dash-bootstrap-templates.dist</li>
    <li>python312Packages.dash.dist</li>
    <li>python312Packages.energyflow</li>
    <li>python312Packages.energyflow.dist</li>
    <li>python312Packages.experiment-utilities</li>
    <li>python312Packages.experiment-utilities.dist</li>
    <li>python312Packages.explorerscript</li>
    <li>python312Packages.explorerscript.dist</li>
    <li>python312Packages.hvplot</li>
    <li>python312Packages.hvplot.dist</li>
    <li>python312Packages.igraph</li>
    <li>python312Packages.igraph.dist</li>
    <li>python312Packages.intake</li>
    <li>python312Packages.intake.dist</li>
    <li>python312Packages.iplotx</li>
    <li>python312Packages.iplotx.dist</li>
    <li>python312Packages.itables</li>
    <li>python312Packages.itables.dist</li>
    <li>python312Packages.kmapper</li>
    <li>python312Packages.kmapper.dist</li>
    <li>python312Packages.leidenalg</li>
    <li>python312Packages.leidenalg.dist</li>
    <li>python312Packages.librosa</li>
    <li>python312Packages.librosa.dist</li>
    <li>python312Packages.neurokit2</li>
    <li>python312Packages.neurokit2.dist</li>
    <li>python312Packages.niaarm</li>
    <li>python312Packages.niaarm.dist</li>
    <li>python312Packages.optuna</li>
    <li>python312Packages.optuna.dist</li>
    <li>python312Packages.piano-transcription-inference</li>
    <li>python312Packages.piano-transcription-inference.dist</li>
    <li>python312Packages.plotly</li>
    <li>python312Packages.plotly.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pyannote-pipeline</li>
    <li>python312Packages.pyannote-pipeline.dist</li>
    <li>python312Packages.pymatgen</li>
    <li>python312Packages.pymatgen.dist</li>
    <li>python312Packages.reflex</li>
    <li>python312Packages.reflex-chakra</li>
    <li>python312Packages.reflex-chakra.dist</li>
    <li>python312Packages.reflex.dist</li>
    <li>python312Packages.resampy</li>
    <li>python312Packages.resampy.dist</li>
    <li>python312Packages.scikit-tda</li>
    <li>python312Packages.scikit-tda.dist</li>
    <li>python312Packages.sumo</li>
    <li>python312Packages.sumo.dist</li>
    <li>python312Packages.synergy</li>
    <li>python312Packages.synergy.dist</li>
    <li>python312Packages.torch-audiomentations</li>
    <li>python312Packages.torch-audiomentations.dist</li>
    <li>python312Packages.torchcrepe</li>
    <li>python312Packages.torchcrepe.dist</li>
    <li>python312Packages.torchlibrosa</li>
    <li>python312Packages.torchlibrosa.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.ax-platform.dist</li>
    <li>python313Packages.chart-studio</li>
    <li>python313Packages.chart-studio.dist</li>
    <li>python313Packages.compressai</li>
    <li>python313Packages.compressai.dist</li>
    <li>python313Packages.dash</li>
    <li>python313Packages.dash-bootstrap-components</li>
    <li>python313Packages.dash-bootstrap-components.dist</li>
    <li>python313Packages.dash-bootstrap-templates</li>
    <li>python313Packages.dash-bootstrap-templates.dist</li>
    <li>python313Packages.dash.dist</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.experiment-utilities.dist</li>
    <li>python313Packages.explorerscript</li>
    <li>python313Packages.explorerscript.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.igraph</li>
    <li>python313Packages.igraph.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.iplotx</li>
    <li>python313Packages.iplotx.dist</li>
    <li>python313Packages.itables</li>
    <li>python313Packages.itables.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.kmapper</li>
    <li>python313Packages.kmapper.dist</li>
    <li>python313Packages.leidenalg</li>
    <li>python313Packages.leidenalg.dist</li>
    <li>python313Packages.librosa</li>
    <li>python313Packages.librosa.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.neurokit2</li>
    <li>python313Packages.neurokit2.dist</li>
    <li>python313Packages.niaarm</li>
    <li>python313Packages.niaarm.dist</li>
    <li>python313Packages.optuna</li>
    <li>python313Packages.optuna.dist</li>
    <li>python313Packages.piano-transcription-inference</li>
    <li>python313Packages.piano-transcription-inference.dist</li>
    <li>python313Packages.plotly</li>
    <li>python313Packages.plotly.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyannote-pipeline</li>
    <li>python313Packages.pyannote-pipeline.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
    <li>python313Packages.resampy</li>
    <li>python313Packages.resampy.dist</li>
    <li>python313Packages.scikit-tda</li>
    <li>python313Packages.scikit-tda.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.synergy</li>
    <li>python313Packages.synergy.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.torch-audiomentations</li>
    <li>python313Packages.torch-audiomentations.dist</li>
    <li>python313Packages.torchcrepe</li>
    <li>python313Packages.torchcrepe.dist</li>
    <li>python313Packages.torchlibrosa</li>
    <li>python313Packages.torchlibrosa.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>theharvester</li>
    <li>theharvester.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>jadx</summary>
<pre>2025-07-11T19:14:36.572+0000 [INFO] [org.gradle.launcher.daemon.server.DaemonRegistryUpdater] Advertising the daemon address to the clients: [04a3dee7-9376-4f59-a1ca-be0d176f2f3b port:50727, addresses:[localhost/127.0.0.1]]
2025-07-11T19:14:36.573+0000 [DEBUG] [org.gradle.launcher.daemon.server.DaemonRegistryUpdater] Advertised daemon context: DefaultDaemonContext[uid=9564c924-19b6-4657-bfba-0c777d61d470,javaHome=/nix/store/grk18frmkshz2ic1r2n0sd7mwscgz8hp-zulu-ca-jdk-21.0.4/Library/Java/JavaVirtualMachines/zulu-21.jdk/Contents/Home,javaVersion=21,javaVendor=Azul Systems, Inc.,daemonRegistryDir=/private/tmp/nix-build-jadx-1.5.0.drv-0/tmp.clhMxJ6rfl/daemon,pid=38058,idleTimeout=120000,priority=NORMAL,applyInstrumentationAgent=true,nativeServicesMode=ENABLED,daemonOpts=-XX:+IgnoreUnrecognizedVMOptions,--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED,--add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED,--add-exports=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED,--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED,--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED,-Dfile.encoding=UTF-8,-Djavax.net.ssl.trustStore=/private/tmp/nix-build-jadx-1.5.0.drv-0/tmp.3oxgDGkGov/keystore,-Djavax.net.ssl.trustStorePassword=3X7VT4K3Q4UEWXBO,-Duser.country=US,-Duser.language=en,-Duser.variant]
2025-07-11T19:14:36.574+0000 [DEBUG] [org.gradle.launcher.daemon.registry.PersistentDaemonRegistry] Storing daemon address: [04a3dee7-9376-4f59-a1ca-be0d176f2f3b port:50727, addresses:[localhost/127.0.0.1]], context: DefaultDaemonContext[uid=9564c924-19b6-4657-bfba-0c777d61d470,javaHome=/nix/store/grk18frmkshz2ic1r2n0sd7mwscgz8hp-zulu-ca-jdk-21.0.4/Library/Java/JavaVirtualMachines/zulu-21.jdk/Contents/Home,javaVersion=21,javaVendor=Azul Systems, Inc.,daemonRegistryDir=/private/tmp/nix-build-jadx-1.5.0.drv-0/tmp.clhMxJ6rfl/daemon,pid=38058,idleTimeout=120000,priority=NORMAL,applyInstrumentationAgent=true,nativeServicesMode=ENABLED,daemonOpts=-XX:+IgnoreUnrecognizedVMOptions,--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED,--add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED,--add-exports=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED,--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED,--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED,-Dfile.encoding=UTF-8,-Djavax.net.ssl.trustStore=/private/tmp/nix-build-jadx-1.5.0.drv-0/tmp.3oxgDGkGov/keystore,-Djavax.net.ssl.trustStorePassword=3X7VT4K3Q4UEWXBO,-Duser.country=US,-Duser.language=en,-Duser.variant]
2025-07-11T19:14:36.581+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire exclusive lock on daemon addresses registry.
2025-07-11T19:14:36.584+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2025-07-11T19:14:36.585+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2025-07-11T19:14:36.586+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire exclusive lock on daemon addresses registry.
2025-07-11T19:14:36.587+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2025-07-11T19:14:36.589+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2025-07-11T19:14:36.590+0000 [LIFECYCLE] [org.gradle.launcher.daemon.server.Daemon] Daemon server started.
2025-07-11T19:14:36.590+0000 [DEBUG] [org.gradle.launcher.daemon.bootstrap.DaemonStartupCommunication] Completed writing the daemon greeting. Closing streams...
2025-07-11T19:14:36.597+0000 [DEBUG] [org.gradle.launcher.daemon.server.Daemon] stopOnExpiration() called on daemon
2025-07-11T19:14:36.612+0000 [DEBUG] [org.gradle.launcher.daemon.server.Daemon] ----- End of the daemon log -----


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.</pre>
</details>
<details>
<summary>python312Packages.optuna-dashboard</summary>
<pre>collecting 17 items                                                            
collecting 89 items                                                            
collecting 127 items                                                           
collecting 217 items                                                           
collected 221 items                                                            

python_tests/artifact/test_backend.py .................                  [  7%]
python_tests/artifact/test_backoff.py ..                                 [  8%]
python_tests/artifact/test_boto3.py ...                                  [  9%]
python_tests/artifact/test_file_system.py ..                             [ 10%]
python_tests/artifact/test_optuna_compatibility.py ..                    [ 11%]
python_tests/artifact/test_prefix.py ..                                  [ 12%]
python_tests/preferential/samplers/test_gp.py ..                         [ 13%]
python_tests/preferential/test_samplers.py .Fatal Python error: Segmentation fault

Thread 0x00000001f2d79f00 (most recent call first):
  File "/nix/store/brw0knlmj41cxn822agj7mvlsl1m4va7-python3.12-torch-2.7.1/lib/python3.12/site-packages/torch/distributions/multivariate_normal.py", line 187 in __init__
  File "/nix/store/qbkvqqdycq76nwx17zjrk7ly3142aai0-python3.12-gpytorch-1.14/lib/python3.12/site-packages/gpytorch/distributions/multivariate_normal.py", line 65 in __init__
  File "/nix/store/6m6ggsz21zk0lmal1bfmjk411wxdpvdh-python3.12-pyro-ppl-1.9.1/lib/python3.12/site-packages/pyro/distributions/distribution.py", line 26 in __call__
/nix/store/0znljckc2qhpsjs033wjc07xzqiysp3g-pytest-check-hook/nix-support/setup-hook: line 23: 77230 Segmentation fault: 11  /nix/store/q54d9qqlb27g2nf9ja4zsa89gl7b3yvm-python3-3.12.11/bin/python3.12 "${flagsArray[@]}"</pre>
</details>
<details>
<summary>python312Packages.wandb</summary>
<pre>  /private/tmp/nix-build-python3.12-wandb-0.19.11.drv-0/source/tests/unit_tests/test_wandb_settings_old.py:33: PytestWarning: Value of environment variable WANDB_CONFIG_DIR type should be str, but got PosixPath('/private/tmp/nix-build-python3.12-wandb-0.19.11.drv-0/pytest-of-_nixbld12/pytest-0/popen-gw0/test__global_path_env_var_exis0/config') (type: PosixPath); converted to str implicitly
    monkeypatch.setenv("WANDB_CONFIG_DIR", config_dir)

tests/unit_tests/test_wandb_settings_old.py::test__global_path_env_var_exists_and_is_writable[wandb_core]
  /private/tmp/nix-build-python3.12-wandb-0.19.11.drv-0/source/tests/unit_tests/test_wandb_settings_old.py:33: PytestWarning: Value of environment variable WANDB_CONFIG_DIR type should be str, but got PosixPath('/private/tmp/nix-build-python3.12-wandb-0.19.11.drv-0/pytest-of-_nixbld12/pytest-0/popen-gw0/test__global_path_env_var_exis1/config') (type: PosixPath); converted to str implicitly
    monkeypatch.setenv("WANDB_CONFIG_DIR", config_dir)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/unit_tests/test_cli.py::test_login_host_trailing_slash_fix_invalid[wandb_core] - assert 1 == 0
 +  where 1 = <Result SystemExit(1)>.exit_code
FAILED tests/unit_tests/test_wandb_settings.py::test_setup_offline[no_wandb_core] - wandb.sdk.service.service.ServiceStartTimeoutError: Timed out waiting for wandb service to start after 30.0 seconds. Try increasing the timeout with the `_service_wait` setting.
FAILED tests/unit_tests/test_wandb_init.py::test_no_root_dir_access__uses_temp_dir[no_wandb_core] - wandb.sdk.service.service.ServiceStartTimeoutError: Timed out waiting for wandb service to start after 30.0 seconds. Try increasing the timeout with the `_service_wait` setting.
FAILED tests/unit_tests/test_cli.py::test_login_bad_host[no_wandb_core-https://app.wandb.ai-did you mean https://api.wandb.ai] - AssertionError: assert 'did you mean https://api.wandb.ai' in '1'
 +  where '1' = str(SystemExit(1))
 +    where SystemExit(1) = <Result SystemExit(1)>.exception
ERROR tests/unit_tests/test_job_builder.py::test_build_repo_notebook_job[no_wandb_core] - wandb.sdk.service.service.ServiceStartTimeoutError: Timed out waiting for wandb service to start after 30.0 seconds. Try increasing the timeout with the `_service_wait` setting.
ERROR tests/unit_tests/test_job_builder.py::test_build_image_job[no_wandb_core] - wandb.sdk.service.service.ServiceStartTimeoutError: Timed out waiting for wandb service to start after 30.0 seconds. Try increasing the timeout with the `_service_wait` setting.
ERROR tests/unit_tests/test_job_builder.py::test_build_artifact_job[no_wandb_core] - wandb.sdk.service.service.ServiceStartTimeoutError: Timed out waiting for wandb service to start after 30.0 seconds. Try increasing the timeout with the `_service_wait` setting.
= 4 failed, 3187 passed, 187 skipped, 8 xfailed, 8 xpassed, 6 warnings, 3 errors in 333.04s (0:05:33) =</pre>
</details>
<details>
<summary>python313Packages.optuna-dashboard</summary>
<pre>plugins: jaxtyping-0.3.2, hypothesis-6.131.17, typeguard-4.4.2
collecting ... 
collecting 0 items                                                             
collecting 127 items                                                           
collected 221 items                                                            

python_tests/artifact/test_backend.py .................                  [  7%]
python_tests/artifact/test_backoff.py ..                                 [  8%]
python_tests/artifact/test_boto3.py ...                                  [  9%]
python_tests/artifact/test_file_system.py ..                             [ 10%]
python_tests/artifact/test_optuna_compatibility.py ..                    [ 11%]
python_tests/artifact/test_prefix.py ..                                  [ 12%]
python_tests/preferential/samplers/test_gp.py ..                         [ 13%]
python_tests/preferential/test_samplers.py .Fatal Python error: Segmentation fault

Thread 0x00000001f2d79f00 (most recent call first):
  File "/nix/store/bjy6ax4pygbyd2p60lpc04yzpnimd91x-python3.13-torch-2.7.1/lib/python3.13/site-packages/torch/distributions/multivariate_normal.py", line 187 in __init__
  File "/nix/store/f6bds88bw0k31y2jmazp9mhvb5kcajlp-python3.13-gpytorch-1.14/lib/python3.13/site-packages/gpytorch/distributions/multivariate_normal.py", line 65 in __init__
  File "/nix/store/fqlck1dzxz234z3dclras7vi8yib9vqf-python3.13-pyro-ppl-1.9.1/lib/python3.13/site-packages/pyro/distributions/distribution.py", line 26 in __call__
  File "/private/tmp/nix-build-python3.13-optuna-dashboard-0.18.0.drv-0/source/optuna_dashboard/preferential/samplers/gp.py"/nix/store/w2k4627hj4xfsz1kwh3bxblv425yw726-pytest-check-hook/nix-support/setup-hook: line 23: 39330 Segmentation fault: 11  /nix/store/j1spz18w5a10r9nj4wpkyh0cx730rs4z-python3-3.13.4/bin/python3.13 "${flagsArray[@]}"</pre>
</details>
</details>

--- a/pkgs/development/python-modules/chart-studio/default.nix
+++ b/pkgs/development/python-modules/chart-studio/default.nix
@@ -49,6 +49,6 @@ buildPythonPackage {
     description = "Utilities for interfacing with Plotly's Chart Studio service";
     homepage = "https://github.com/plotly/chart-studio";
     license = with lib.licenses; [ mit ];
-    maintainers = [ ];
+    maintainers = with lib.maintainers; [ sarahec ];
   };
 }

--- a/pkgs/development/python-modules/chart-studio/default.nix
+++ b/pkgs/development/python-modules/chart-studio/default.nix
@@ -10,21 +10,17 @@
   pytestCheckHook,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "chart-studio";
-  version = "1.1.0-unstable-2024-07-23";
+  version = "1.1.0-unstable-2025-01-30";
   pyproject = true;
 
-  # chart-studio was split from plotly
   src = fetchFromGitHub {
     owner = "plotly";
-    repo = "plotly.py";
-    # We use plotly's upstream version as it's the same repo, but chart studio has its own version number.
-    rev = "v5.23.0";
-    hash = "sha256-K1hEs00AGBCe2fgytyPNWqE5M0jU5ESTzynP55kc05Y=";
+    repo = "chart-studio";
+    rev = "44c7c43be0fe7e031ec281c86ee7dae0efa0619e";
+    hash = "sha256-RekcZzUcunIqXOSriW+RvpLdvATQWTeRAiR8LFodfQg=";
   };
-
-  sourceRoot = "${src.name}/packages/python/chart-studio";
 
   build-system = [ setuptools ];
 
@@ -36,6 +32,7 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [
     mock
+    plotly
     pytestCheckHook
   ];
 
@@ -46,12 +43,11 @@ buildPythonPackage rec {
   # most tests talk to a network service, so only run ones that don't do that.
   pytestFlagsArray = [
     "chart_studio/tests/test_core"
-    "chart_studio/tests/test_plot_ly/test_api"
   ];
 
   meta = {
     description = "Utilities for interfacing with Plotly's Chart Studio service";
-    homepage = "https://github.com/plotly/plotly.py/tree/master/packages/python/chart-studio";
+    homepage = "https://github.com/plotly/chart-studio";
     license = with lib.licenses; [ mit ];
     maintainers = [ ];
   };

--- a/pkgs/development/python-modules/plotly/default.nix
+++ b/pkgs/development/python-modules/plotly/default.nix
@@ -38,14 +38,14 @@
 
 buildPythonPackage rec {
   pname = "plotly";
-  version = "6.1.2";
+  version = "6.2.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "plotly";
     repo = "plotly.py";
     tag = "v${version}";
-    hash = "sha256-+vIq//pDLaaTmRGW+oytho3TfMmLCtuIoHeFenLVcek=";
+    hash = "sha256-Vfj5jG0AkBjivExOx7oMoocTopWl0yMc1INpEbtlgTc=";
   };
 
   postPatch = ''
@@ -105,27 +105,34 @@ buildPythonPackage rec {
 
   __darwinAllowLocalNetworking = true;
 
-  disabledTestPaths = lib.optionals stdenv.hostPlatform.isDarwin [
-    # fails to launch kaleido subprocess
-    "tests/test_optional/test_kaleido"
-    # numpy2 related error, RecursionError
-    # See: https://github.com/plotly/plotly.py/issues/4852
-    "tests/test_plotly_utils/validators/test_angle_validator.py"
-    "tests/test_plotly_utils/validators/test_any_validator.py"
-    "tests/test_plotly_utils/validators/test_color_validator.py"
-    "tests/test_plotly_utils/validators/test_colorlist_validator.py"
-    "tests/test_plotly_utils/validators/test_colorscale_validator.py"
-    "tests/test_plotly_utils/validators/test_dataarray_validator.py"
-    "tests/test_plotly_utils/validators/test_enumerated_validator.py"
-    "tests/test_plotly_utils/validators/test_fig_deepcopy.py"
-    "tests/test_plotly_utils/validators/test_flaglist_validator.py"
-    "tests/test_plotly_utils/validators/test_infoarray_validator.py"
-    "tests/test_plotly_utils/validators/test_integer_validator.py"
-    "tests/test_plotly_utils/validators/test_number_validator.py"
-    "tests/test_plotly_utils/validators/test_pandas_series_input.py"
-    "tests/test_plotly_utils/validators/test_string_validator.py"
-    "tests/test_plotly_utils/validators/test_xarray_input.py"
-  ];
+  disabledTestPaths =
+    [
+      # Broken imports
+      "plotly/matplotlylib/mplexporter/tests"
+      # Fails to catch error when serializing document
+      "tests/test_optional/test_kaleido/test_kaleido.py::test_defaults"
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      # fails to launch kaleido subprocess
+      "tests/test_optional/test_kaleido"
+      # numpy2 related error, RecursionError
+      # See: https://github.com/plotly/plotly.py/issues/4852
+      "tests/test_plotly_utils/validators/test_angle_validator.py"
+      "tests/test_plotly_utils/validators/test_any_validator.py"
+      "tests/test_plotly_utils/validators/test_color_validator.py"
+      "tests/test_plotly_utils/validators/test_colorlist_validator.py"
+      "tests/test_plotly_utils/validators/test_colorscale_validator.py"
+      "tests/test_plotly_utils/validators/test_dataarray_validator.py"
+      "tests/test_plotly_utils/validators/test_enumerated_validator.py"
+      "tests/test_plotly_utils/validators/test_fig_deepcopy.py"
+      "tests/test_plotly_utils/validators/test_flaglist_validator.py"
+      "tests/test_plotly_utils/validators/test_infoarray_validator.py"
+      "tests/test_plotly_utils/validators/test_integer_validator.py"
+      "tests/test_plotly_utils/validators/test_number_validator.py"
+      "tests/test_plotly_utils/validators/test_pandas_series_input.py"
+      "tests/test_plotly_utils/validators/test_string_validator.py"
+      "tests/test_plotly_utils/validators/test_xarray_input.py"
+    ];
 
   pythonImportsCheck = [ "plotly" ];
 

